lib/plugins/helper/feed_tag.js

```diff
@@ -1,9 +1,36 @@
 'use strict';
 
 function feedTagHelper(path, options = {}) {
-  const title = options.title || this.config.title;
+  const { config } = this;
+  const title = options.title || config.title;
+  const { feed } = config;
 
-  return `<link rel="alternate" href="${this.url_for(path)}" title="${title}">`;
+  if (path) {
+    if (typeof path !== 'string') throw new TypeError('path must be a string!');
+    const { type } = options;
+    let typeAttr = '';
+
+    if (type === 'atom' || path.includes('atom')) typeAttr = 'type="application/atom+xml"';
+    else if (type.includes('rss') || path.includes('rss')) typeAttr = 'type="application/rss+xml"';
```

Comment:
there is an edge case, `feed_tag('feed.xml')` and `type` is not specified; `typeAttr` will be empty as "feed.xml" can refer to atom or rss.

It's fine to leave it empty since currently this helper doesn't add `type="application/atom+xml"`.